src/cairoUtilFuncGen/storage/mappingIndexAccess.ts

```diff
@@ -60,8 +91,12 @@ export class MappingIndexAccessGen extends StringIndexedFuncGen {
       return existing.name;
     }
 
-    const funcName = `WS${this.generatedFunctions.size}_INDEX_${indexType.typeName}_to_${valueType.typeName}`;
-    const mappingName = `WARP_MAPPING${this.generatedFunctions.size}`;
+    const funcName = `WS${this.generatedFunctions.size - this.generatedHashFunctionNumber}_INDEX_${
+      indexType.typeName
+    }_to_${valueType.typeName}`;
+    const mappingName = `WARP_MAPPING${
+      this.generatedFunctions.size - this.generatedHashFunctionNumber
```

Comment:
Can you explain what this is for?

Reply:
I wanted to distinguish the numbers given as postfix to the mapping functions and hashing functions. So since it already has `WARP_MAPPING0...`, the hash functions were automatically labeled as `ws_string_hash1`. So I separated the numbers of mapping functions and hash functions